mks/model/stuff.py

```diff
@@ -289,23 +313,60 @@ def get_kinderen(self):
             {'name': 'geslachtsnaam', 'parser': self.to_string},
             {'name': 'geslachtsaanduiding', 'parser': self.to_string},
             {'name': 'geboortedatum', 'parser': self.to_date},
+            {'name': 'inp.geboorteplaats', 'parser': self.to_date, 'save_as': 'geboorteplaats'},
+            {'name': 'inp.geboorteLand', 'parser': self.to_date, 'save_as': 'geboorteLand'},
             {'name': 'overlijdensdatum', 'parser': self.to_date},
+            {'name': 'adellijkeTitelPredikaat', 'parser': self.to_date},
```

Comment:
Same here, shouldn't this be `self.to_string` ?

Reply:
Copy pasta mistake :see_no_evil: